### PR TITLE
feat: default what to expect text and additional what to expect field

### LINKED ISSUE #1054: Detroit: What to Expect default + view + script
**Acceptance Criteria**
* When I am on a public listing page, then I will see the What to Expect section 
* When I am in partners, I expect to see the default text. And I can edit, even if that messes with the formatting 

**Notes / Questions**
* Ideally we can keep the formatting, not sure the best way to do that 
* There's a read more
* For existing listings, we should script in the copy to all of them
* For new listings, this should be the default 

**Copy** 
Vacant Units:
If you are looking to move in immediately, contact the property and ask if they have any vacant units. 

Waitlists:
If none are vacant, but you are still interested in living at the property in the future, ask how you can be placed on their waitlist. 

* Property staff should walk you through the process to get on their waitlist. 
* You can be on waitlists for multiple properties, but you will need to contact each one of them to begin that process.
* Even if you are on a waitlist, it can take months or over a year to get an available unit for that building.
* Many properties that are affordable because of government funding or agreements have long waitlists. If you're on a waitlist for a property, you will be notified as available units come up.

**Designs** 
https://www.figma.com/file/D2fNPS5Ucll8uz7hyMUQGB/bloom-public-single-listing?node-id=1238%3A19373

image

image

### PR DESCRIPTION
**Pull request by emilyjablonski**
## Issue

- Closes #1054

## Description

Add a new `whatToExpect` text default, and the ability to add a read more section to the what to expect text which lives in the new `whatToExpectAdditionalText` field.

## How Can This Be Tested/Reviewed?

Ensure the default is set on new listings and that existing listings all have their what to expect and additional information fields set to the defaults.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: ui-components/src/page_components/listing/listing_sidebar/events/OpenHouseEvent.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const OpenHouseEvent = (props: { events: ListingEvent[] }) => {
         <div key={`openHouses-${index}`}>
           <EventDateSection event={openHouseEvent} />
           {openHouseEvent.url && (
-            <p className="text text-gray-800 pb-3">
+            <p className="text text-gray-807500 pb-3">
```
Comment (jaredcwhite) [inline ui-components/src/page_components/listing/listing_sidebar/events/OpenHouseEvent.tsx:14]: Is this a typo?

File: ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
   const listing = props.listing
-
+  if (!listing.whatToExpect || listing.whatToExpect === "") return null
   return (
     <section className="aside-block">
       <h4 className="text-caps-underline">{t("whatToExpect.label")}</h4>
-      <p className="text-tiny text-gray-800">
-        {listing.whatToExpect ? listing.whatToExpect : t("whatToExpect.default")}
-      </p>
+      <div className="text-tiny text-gray-750">
```
Comment (jaredcwhite) [inline ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx:17]: In the interests of future sync'ing of `ui-components` drift, do you think this change is written in a such a way we'd be able to  backport it to core?

**Review by YazeedLoonat — CHANGES_REQUESTED** [review]
Hey @emilyjablonski , outcome looks good, I think there is an issue with the migration script and what it does when doing a full reseed though. Right now if you run a db:reseed:detroit the newly seeded listing don't have what to expect text

**Comment by emilyjablonski** [comment]
Sorry all this is still a `wip`

File: ui-components/src/page_components/listing/listing_sidebar/events/OpenHouseEvent.tsx (hunk 1 above)
Comment (emilyjablonski) [reply ui-components/src/page_components/listing/listing_sidebar/events/OpenHouseEvent.tsx]: Yes, thank you!

File: ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx (hunk 2 above)
Comment (emilyjablonski) [reply ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx:16]: Updated 👍

File: sites/partners/src/listings/PaperListingForm/formTypes.ts [hunk 3]
```diff
@@ -143,8 +143,8 @@ export const formDefaults: FormListing = {
...
+  whatToExpectAdditionalText: `<ul className="list-disc pl-6"><li>Property staff should walk you through the process to get on their waitlist.</li><li>You can be on waitlists for multiple properties, but you will need to contact each one of them to begin that process.</li><li>Even if you are on a waitlist, it can take months or over a year to get an available unit for that building.</li><li>Many properties that are affordable because of government funding or agreements have long waitlists. If you're on a waitlist for a property, you will be notified as available units come up.</li></ul>`,
+  whatToExpect: `<div><strong>Vacant Units</strong>:<div className="mb-3">If you are looking to move in immediately, contact the property and ask if they have any vacant units.</div><div><strong>Waitlists</strong>:<div>If none are vacant, but you are still interested in living at the property in the future, ask how you can be placed on their waitlist.</div>`,
```
Comment (seanmalbert) [inline sites/partners/src/listings/PaperListingForm/formTypes.ts:147]: We really should introduce a WYSIWYG editor to handle this text better. Leaving this comment here to spin off a new issue.

**Review by seanmalbert — CHANGES_REQUESTED** [review]

File: backend/core/src/migration/1649893064530-add-what-to-expect-additional-text.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    const defaultWhatToExpectAdditionalText = `<ul className="list-disc pl-6"><li>Property staff should walk you through the process to get on their waitlist.</li><li>You can be on waitlists for multiple properties, but you will need to contact each one of them to begin that process.</li><li>Even if you are on a waitlist, it can take months or over a year to get an available unit for that building.</li><li>Many properties that are affordable because of government funding or agreements have long waitlists. If you're on a waitlist for a property, you will be notified as available units come up.</li></ul>`
+    await queryRunner.query(`ALTER TABLE "listings" ADD "what_to_expect_additional_text" text`)
+
+    await queryRunner.query(`UPDATE listings SET what_to_expect = ($1)`, [defaultWhatToExpect])
+    await queryRunner.query(`UPDATE listings SET what_to_expect_additional_text = ($1)`, [
+      defaultWhatToExpectAdditionalText,
+    ])
+  }
```
[the remark this answers is not in the document]
Comment (emilyjablonski) [reply backend/core/src/migration/1649893064530-add-what-to-expect-additional-text.ts:15]: Yes we want to add it to all existing listings (noted in the issue)

**Review by seanmalbert — APPROVED** [review]
LGTM after response to question around setting what to expect.

**Comment by seanmalbert** [comment]
@YazeedLoonat , can you re-review, please.

File: ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx [hunk 5]
```diff
@@ -1,20 +1,28 @@
...
-  listing: Listing
+  content: string
+  expandableContent: string
 }
 
-const WhatToExpect = (props: WhatToExpectProps) => {
-  const listing = props.listing
-
+const WhatToExpect = ({ content, expandableContent }: WhatToExpectProps) => {
+  if (!content || content === "") return null
```
Comment (YazeedLoonat) [inline ui-components/src/page_components/listing/listing_sidebar/WhatToExpect.tsx:12]:
```suggestion
  if (!content) return null
```

**Review by YazeedLoonat — APPROVED** [review]
LGTM

**Pull request merged**